### PR TITLE
fix Access::can_write

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by Emilgardis** [comment]
What is 15a33cfb4a3bb86b94733f47195c568994c77389 supposed to do?

File: svd-rs/CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v0.11.0] - 2021-10-02
```
Comment (Emilgardis) [inline svd-rs/CHANGELOG.md:8]: This should probably be 0.11.1

Comment (Emilgardis) [reply svd-rs/CHANGELOG.md:8]: Also needs a bump version in Cargo.toml

Comment (burrbull) [reply svd-rs/CHANGELOG.md]: Sorry. Fixed

**Pull request merged**